aten/src/ATen/native/cuda/RowwiseScaledMM.cu

```diff
@@ -174,7 +174,7 @@ void f8f8bf16_rowwise_impl(
 
   // Implement rowwise scaling epilogue.
   constexpr int ColBroadcastStages = 0;
-  constexpr int RowBroadcastStages = PingPong::value ? 2 : 1;
+  constexpr int RowBroadcastStages = 0;
```

Comment:
NOTE: This change is required to compile, but I have "misaligned address" errors after that when I use CUTLASS 3.6 and use the PingPong kernels. I found a workaround by inverting the order of the scales, eg:
```c++
  using EpilogueEVT = cutlass::epilogue::fusion::Sm90EVT<Cast,
      cutlass::epilogue::fusion::Sm90EVT<Add, Bias,
        cutlass::epilogue::fusion::Sm90EVT<Multiply, WScale, 
          cutlass::epilogue::fusion::Sm90EVT<Multiply, XScale,
            Accum>>>>;
```

(requires to change how the arguments are supplied to the epilogue as well)
cc @drisspg